docs/migrating-configs.md

```diff
@@ -157,6 +157,99 @@ IP addresses and domain names can also include a port number (`1.2.3.4:80`).
   }
 }
 ```
+## From Version 3.1.0 to 3.2.0
+
+There are not any breaking changes between versions 3.1.0 and 3.2.0 of the configuration specification. Any valid 3.1.0 configuration can be updated to a 3.2.0 configuration by simply changing the version string in the config.
+
+The following is a list of notable new features, deprecations, and changes.
+
+### Partition Resizing
+
+The `partition` section gained a new `resize` field. When specified Ignition will resize an existing partition if it matches the config in all respects except the partition size. Refer to the [operator-notes][operator-notes] for more information.
+
+```json
+{
+  "ignition": { "version": "3.2.0" },
+  "storage": {
+    "disks": [{
+      "device": "/dev/sda",
+      "partitions": [{
+        "label": "root",
+        "sizeMiB": 16384,
+        "resize": "true"
+      }]
+    }]
+  }
+}
+```
+
+### LUKS
+
+Ignition now supports creating LUKS2 devices. Refer to the [operator-notes][operator-notes] for more information.
+
+```json
+{
+  "ignition": {"version": "3.2.0"},
+  "storage": {
+    "luks": [{
+      "name": "data",
+      "device": "/dev/sdb"
+    }],
```

Comment:
Won't this config create an ephemeral encrypted volume?  That's not a common operation, and since the data won't survive a reboot, we probably shouldn't give it as the canonical example of how to use this feature.